tests/sentry/ingest/ingest_consumer/test_ingest_consumer_processing.py

```diff
@@ -161,6 +161,86 @@ def test_transactions_spawn_save_event_transaction(
     )
 
 
+@django_db_all
+def test_transactions_save_event_transaction_is_tracked(default_project):
```

Comment:
What are these tests actually testing? That true just logs a line and false does not. Is that actually worthwhile, especially if the sampling function is mocked anyway?

Reply:
i think still worth? Because it's testing in the `process_event` function, logging is getting called or not for a transaction

Reply:
I think you should make `event_tracker` generic so it can be used in all ingest pipelines, and tests for the `event_tracker`.

It shouldn't be the responsibility of the pipelines that will use this module to write the tests.

Reply:
done